Backend/Backend/Controllers/Requests/ProfileRequests.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Backend.Controllers.Requests
+{
+   [BindProperties]
+   public class ChangePasswordRequest
+   {
+      [Required]
+      public string OldPassword { get; set; }
+      
+      [Required]
+      public string NewPassword { get; set; }
+      
+      [Required]
+      public string ConfirmNewPassword { get; set; }
```

Comment:
Shouldnt this be handled in the frontend?

Reply:
Nothing wrong with redundancy